src/test/scala/tech/sourced/gemini/BaseSparkSpec.scala

```diff
@@ -2,32 +2,50 @@ package tech.sourced.gemini
 
 import org.apache.spark.SparkConf
 import org.apache.spark.sql.SparkSession
-import org.scalatest.{BeforeAndAfterAll, Suite}
+import org.scalatest.{BeforeAndAfterAll, Suite, Tag}
 
 trait BaseSparkSpec extends BeforeAndAfterAll {
   this: Suite =>
 
-  @transient var sparkSession: SparkSession = _
+  var sparkSession: SparkSession = _
   private var _conf: SparkConf = _
 
   def useSparkConf(conf: SparkConf): SparkConf = {
     _conf = conf
     _conf
   }
 
+  def useDefaultSparkConf(): SparkConf = {
+    val defaultConf: SparkConf = new SparkConf(true)
+      .setAppName(this.getClass.getSimpleName)
+      .set("spark.cassandra.connection.host", Gemini.defaultCassandraHost)
+      .set("spark.cassandra.connection.port", Gemini.defaultCassandraPort.toString)
+      .set("spark.cassandra.connection.keep_alive_ms", "5000")
+      .set("spark.cassandra.connection.timeout_ms", "30000")
+      .set("spark.ui.showConsoleProgress", "false")
+      .set("spark.ui.enabled", "false")
+      .set("spark.cleaner.ttl", "3600")
+
+    useSparkConf(defaultConf)
+  }
+
   override protected def beforeAll(): Unit = {
     super.beforeAll()
 
     sparkSession = SparkSession.builder()
-      .appName("test").master("local[*]")
```

Comment:
I find it useful, in case Apache Spark is configued to preserve history, to be able to identify sessions that belong to tests, and simplest way to do it in WebUI is by name. 

May be there can be a better name, but do you see any particular reason to remove this?

Reply:
there is `setAppName(this.getClass.getSimpleName)` in `useDefaultSparkConf` now.
It will produce different names for different test suites.

And in general set the name in config looks cleaner.